presto-jdbc/src/main/java/com/facebook/presto/jdbc/PrestoDriverUri.java

```diff
@@ -149,6 +152,24 @@ public Properties getProperties()
         return properties;
     }
 
+    public Optional<String> getTimeZoneID()
+            throws SQLException
+    {
+        Optional<String> timezone = TIMEZONE_ID.getValue(properties);
+
+        if (timezone.isPresent()) {
+            try {
+                // Validate timezone and return it.
+                ZoneId.of(timezone.get());
+                return timezone;
+            }
+            catch (DateTimeException e) {
+                throw new SQLException("Specified TimeZoneID is not supported: " + timezone.get(), e);
+            }
+        }
```

Comment:
**Suggestion**: Since we are not using `ZoneId`, we can probably validate as follows: (NIT: also updated the parameter name `timeZoneID` to match with actual name in the exception message)

```suggestion
        if (timezone.isPresent()) {
            List<String> timeZoneIds = Arrays.asList(TimeZone.getAvailableIDs());
            if (!timeZoneIds.contains(timezone.get())) {
                throw new SQLException("Specified timeZoneID is not supported: " + timezone.get());
            }
            return timezone.get();
        }
```